recipe/meta.yaml

```diff
@@ -48,3 +48,5 @@ extra:
   recipe-maintainers:
     - mvanniekerk
```

Comment:
np. Honestly I copy mine from my profile just to avoid potential issues like this.